src/elements/LayoutDashboard.tsx

```diff
@@ -1,79 +1,129 @@
+        <DashboardMenu isOpen={isOpen} onClose={onClose} reduced={reduced} />
+
+        {/* Main Content */}
+        <Flex flex='1 1 0' flexDirection={'column'}>
+          {/* Top Menu */}
+          <Box
+            position={'sticky'}
+            bgColor='white'
+            top={0}
+            px={4}
+            gap={4}
+            display='flex'
+            h={16}
+            flexShrink={0}
+            alignItems='center'
+            borderBottom='var(--border)'
+          >
             <IconButton
-              as={ReactRouterLink}
-              to={back}
-              onClick={() => {
-                setBack(null)
-                setTitle(null)
-              }}
-              aria-label='Back'
-              icon={<MdKeyboardArrowLeft />}
-              colorScheme='brand'
-              color='black'
+              icon={<LayoutRight />}
+              aria-label='Open menu'
+              variant={'transparent'}
+              colorScheme='gray'
+              size={'xs'}
+              onClick={isMobile ? onOpen : () => setReduced((prev) => !prev)}
             />
-          )}
-          {title && (
-            <Heading size={'sm'} fontWeight={500} alignSelf='center' mb={0} pl='10px' fontSize='1.5rem'>
-              {title}
-            </Heading>
-          )}
-          <Box ml='auto' display='flex' gap={3} alignItems='center'>
-            {/* User profile & menu */}
-            <AccountMenu />
-            {/* Hamburger button to open sidebar on small screens */}
-            <IconButton icon={<HamburgerIcon />} onClick={onOpen} display={{ xl: 'none' }} aria-label='Open menu' />
-          </Box>
-        </Box>
 
-        {/* Sidebar for large screens */}
-        <DashboardMenu isOpen={isOpen} onClose={onClose} />
+            <Box borderRight={'var(--border)'} h={6} />
 
-        {/* Main Content */}
-        <Box gridArea='main' mr={{ base: 0, xl: 6 }} mb={4} minW={0}>
-          <Outlet context={{ setTitle, setBack } satisfies DashboardLayoutContext} />
-        </Box>
-      </Grid>
+            <Box as='nav'>
+              <OrderedList display={'flex'} alignItems={'center'} gap={1.5} styleType={"''"} ml={0}>
+                {!!breadcrumb.length ? (
+                  <>
+                    <ListItem>
+                      <Link
+                        as={ReactRouterLink}
+                        to={generatePath(Routes.dashboard.base)}
+                        variant={'breadcrumb'}
+                        onClick={() => setBreadcrumb([])}
+                      >
+                        {t('organization.dashboard')}
+                      </Link>
+                    </ListItem>
+                    <ListItem display={'flex'} justifyContent={'center'} alignItems={'center'}>
+                      <Icon as={ChevronRight} />
+                    </ListItem>
+                  </>
+                ) : (
+                  <Text>{t('organization.dashboard')}</Text>
+                )}
```

Comment:
It doesn't feel ok that the layout is different when is linked compared when it's not. And Ordered (or Unordered) List must have ListItem as children, so this is not correct. The `Text` should be (or be wrapped with) a ListItem too.